tests/test_queue.py

```diff
@@ -435,9 +434,8 @@ async def test_sweep(self, mock_time: MagicMock) -> None:
         job3 = await self.enqueue("test", timeout=1)
         for _ in range(4):
             job = await self.dequeue()
-            job.status = Status.ACTIVE
             job.started = 1000
```

Comment:
Bit awkward that there are two different ways to update the job - setting attributes or kwargs.